### PR TITLE
Use a local verifier call for issuer tests.

### PR DESCRIPTION
**Pull request by tminard**

### CODE REVIEW

File: tests/vc-verifier/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
```
Comment (tminard) [inline tests/vc-verifier/index.js:4]: @BigBlueHat I copied the current file header but LMK if this repo license needs to be changed

**Review by BigBlueHat — APPROVED** [review]
One small tweak on the copyright year--but it's non-blocking, so approving. Great work, @tminard!

File: tests/assertions.js [hunk 2]
```diff
@@ -77,7 +77,10 @@ export const verificationSuccess = async ({credential, verifier}) => {
       checks: ['proof']
     }
   };
-  const {result, error} = await verifier.post({json: body});
+  const results = await verifier.post({json: body});
+
+  const {result, error} = results;
```
Comment (aljones15) [inline tests/assertions.js:82]: why not just stick with the original destructuring?

File: tests/vc-verifier/index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import {cryptosuite} from '@digitalbazaar/ecdsa-rdfc-2019-cryptosuite';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {didIo} from '@bedrock/did-io';
+import {documentLoader} from '../vc-generator/documentLoader.js';
+
+// Configure the driver for ECDSA DID keys
+const driver = didKey.driver();
+
+driver.use({
+  multibaseMultikeyHeader: 'zDna',
```
Comment (aljones15) [inline tests/vc-verifier/index.js:18]: does this work with ecdsa p-384 keys which have a different header?

**Review by aljones15 — APPROVED** [review]
This is pretty impressive and well written. p-384 support is my only real concern.

File: tests/assertions.js (hunk 2 above)
Comment (tminard) [reply tests/assertions.js:82]: Yeah I can put that back - this is a side effect of changes I made while debugging

Comment (aljones15) [reply tests/assertions.js:82]: yeah I had a feeling you logged results to see what's inside of it. code for that is partially here: https://github.com/w3c/vc-test-suite-implementations/blob/d038e90dd7581c435b7dbada3e3a43d7aa4886c9/lib/requests.js#L60

There's a second one for zcap calls after it.

File: package.json [hunk 4]
```diff
@@ -50,6 +50,7 @@
     "@digitalbazaar/multikey-context": "^2.0.1",
     "@digitalbazaar/vc": "github:digitalbazaar/vc#vc-2.0-time-props",
     "@digitalcredentials/did-context": "^1.0.0",
+    "@bedrock/did-io": "^10.0.0",
```
Comment (aljones15) [inline package.json:53]: one small thing here. I'm assuming this works, but the `@bedrock` libs are designed to work inside of a bedrock instance. The test suites are 100% not bedrock. You could try using the dependent library: https://github.com/digitalbazaar/bedrock-did-io/blob/add41972383ddb0c265c9ecd55e2b41bec99809b/package.json#L31

`@digitalbazaar/did-io`

File: tests/10-rdfc-create.js [hunk 5]
```diff
@@ -5,18 +5,17 @@
 import {
   shouldBeBs58, shouldBeMulticodecEncoded, verificationSuccess
 } from './assertions.js';
-import chai from 'chai';
 import {createInitialVc} from './helpers.js';
 import {documentLoader} from './documentLoader.js';
 import {endpoints} from 'vc-test-suite-implementations';
 import {getSuiteConfig} from './test-config.js';
+import {LocalVerifier} from './vc-verifier/index.js';
```
Comment (aljones15) [inline tests/10-rdfc-create.js:12]: usually you only capitalize es6 classes or factory functions. so this should probably be `localVerifier`.

File: tests/vc-verifier/index.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  return documentLoader(url);
+}
+
+/**
+ * Local instance of a Verifier service that can be used to verify VCs locally.
+ *
+ * This is not intended to be a complete implementation but sufficient for local
+ * testing only.
```
Comment (TallTed) [inline tests/vc-verifier/index.js:58]:
```suggestion
 * This is not intended to be a complete implementation but sufficient only for
 * local testing.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
small, human-facing, for clarity

**Pull request merged**